Classes/Controller/ProductController.php

```diff
@@ -8,7 +8,11 @@
  * For the full copyright and license information, please read the
  * LICENSE file that was distributed with this source code.
  */
-
+use Psr\Http\Message\ResponseInterface;
```

Comment:
Can you change the order of the imports? My php-cs-fixer claims about:

   1) cart_products/Build/../Classes/Controller/ProductController.php (ordered_imports)
   2) cart_products/Build/../Classes/Utility/ProductUtility.php (ordered_imports)
   3) cart_products/Build/../Classes/ViewHelpers/CanonicalTagViewHelper.php (ordered_imports)
   4) cart_products/Build/../Classes/Domain/Repository/Product/ProductRepository.php (ordered_imports)
   5) cart_products/Build/../Classes/Domain/Repository/CategoryRepository.php (single_line_after_imports)
   6) cart_products/Build/../Classes/Domain/Model/Product/Product.php (ordered_imports)
   7) cart_products/Build/../Classes/Domain/Model/Product/BeVariant.php (ordered_imports)
   8) cart_products/Build/../Classes/Domain/Model/Category.php (single_line_after_imports, ordered_imports)